### PR TITLE
Feature/extra features

### PR DESCRIPTION
**Pull request by HanaNavratilova**

### CODE REVIEW

File: src/utils/getKeys.ts [hunk 1]
```diff
@@ -7,5 +7,5 @@ const getArgumentsArray = (...keys: Uuid[]): Uuid[] => keys;
 // Set max to sth means that cache size is limited.
 const memoizeKeys = memoize(getArgumentsArray, {length: false, max: 1});
 
-export const getKeys = (list: IListItem[]): Uuid[] =>
-  memoizeKeys(...list.map((item: ListItem) => item.id));
+export const getKeys = (list: object[]): Uuid[] =>
```
Comment (medvedovic) [inline src/utils/getKeys.ts]: u sure, u want to accept `object[]`? 😄

File: src/utils/sortItems.ts [hunk 2]
```diff
@@ -5,14 +5,23 @@ import { ListItem } from '../models/ListItem';
...
+    switch (sorting) {
+      case ListSorting.CreatedTime:
+        return compareTwoTimes(left.creationTime, right.creationTime);
+      case ListSorting.LastUpdateTime:
+        return compareTwoTimes(left.lastUpdateTime, right.lastUpdateTime);
+      default:
+        throw new Error('Unknown sorting type');
+    }
+  };
+
```
Comment (medvedovic) [inline src/utils/sortItems.ts]: 🚨 copy-pasta alert 😛

Comment (HanaNavratilova) [reply src/utils/sortItems.ts]: 😇

File: src/components/List.tsx [hunk 3]
```diff
@@ -3,36 +3,84 @@ import * as PropTypes from 'prop-types';
...
+  render(): JSX.Element {
+    return (
+      <div className="row">
+        <div className="col-sm-12 col-md-16">
+          <ul className="nav nav-tabs border-info">
+            <li className="nav-item">
+              <div
+                className={this.props.sorting === ListSorting.CreatedTime ? 'nav-link active border-info border-bottom-0 text-info' : 'nav-link'}
+                onClick={this.changeViewToCreatedTime}
+              >Created time
```
Comment (medvedovic) [inline src/components/List.tsx]: Always put the content (children, if you want) on a new line

File: src/constants/ListSorting.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export enum ListSorting {
+  CreatedTime = 'CreatedTime',
+  LastUpdateTime = 'LastUpdateTime'
+}
+
+export const getListSortingArray = (): ListSorting[] =>  [ListSorting.CreatedTime, ListSorting.LastUpdateTime];
```
[the remark this answers is not in the document]
Comment (PetrSvirak) [reply src/constants/ListSorting.ts]: To clarify the subtle irony above: Nothing actually prevents us from using `ES2017` - it just a little bit o [babel setup](https://babeljs.io/docs/en/babel-preset-es2017) and package meddling here and there 😛

**Pull request merged**